contrib/completion/bash/docker

```diff
@@ -469,7 +477,7 @@ _docker_images() {
 
 	case "$cur" in
 		-*)
-			COMPREPLY=( $( compgen -W "--all -a --filter -f --help --no-trunc --quiet -q" -- "$cur" ) )
+			COMPREPLY=( $( compgen -W "--all -a --digests=true --filter -f --help --no-trunc --quiet -q" -- "$cur" ) )
```

Comment:
Shouldn't this just be `--digests`? (since it's a boolean flag and it defaults to `false`, so it should work the same as `--all` which is also a boolean flag)